<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Demo</class>
 <widget class="QMainWindow" name="Demo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1244</width>
    <height>941</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtShanoir-demo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>2</horstretch>
     <verstretch>2</verstretch>
    </sizepolicy>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>2</width>
     <height>2</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,0,0" columnstretch="4,1" rowminimumheight="0,0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item row="2" column="0">
       <widget class="QtShanoirTreeWidget" name="treeWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>2</horstretch>
          <verstretch>2</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>2</width>
          <height>2</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="3" column="0">
         <widget class="QtShanoirDownloadWidget" name="downloadWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>71</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>235</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QtShanoirDetailsWidget" name="detailsWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>71</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>200</height>
           </size>
          </property>
          <property name="title">
           <string>Server Configuration :</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QtShanoirSettingsWidget" name="settingsWidget" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>222</width>
               <height>114</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>350</width>
               <height>152</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>9</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="downloadButton">
            <property name="maximumSize">
             <size>
              <width>350</width>
              <height>27</height>
             </size>
            </property>
            <property name="text">
             <string>Download</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="uploadButton">
            <property name="maximumSize">
             <size>
              <width>350</width>
              <height>27</height>
             </size>
            </property>
            <property name="text">
             <string>Upload</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="SearchFilterBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>150</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>1</width>
          <height>1</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="title">
         <string>Search Filter :</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QtShanoirSearchWidget" name="searchWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>4</horstretch>
             <verstretch>4</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>71</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>71</height>
            </size>
           </property>
           <property name="sizeIncrement">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QtShanoirErrorWidget" name="errorWidget" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>31</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="findButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>67</width>
             <height>27</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>10000</width>
             <height>10000</height>
            </size>
           </property>
           <property name="text">
            <string>Find</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/Images/search.22x22.png</normaloff>:/Images/search.22x22.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1244</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtShanoirTreeWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtShanoirTreeWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtShanoirSettingsWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtShanoirSettingsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtShanoirDetailsWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtShanoirDetailsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtShanoirSearchWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtShanoirSearchWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtShanoirDownloadWidget</class>
   <extends>QWidget</extends>
   <header>qtshanoirdownloadwidget.h</header>
  </customwidget>
  <customwidget>
   <class>QtShanoirErrorWidget</class>
   <extends>QWidget</extends>
   <header>qtshanoirerrorwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
